python/ray/_private/ray_option_utils.py

```diff
@@ -109,96 +112,6 @@ def _validate_resources(resources: Optional[Dict[str, float]]) -> Optional[str]:
     return None
 
 
-def _maybe_warn_misconfigured_tpu_chips(resources: Dict[str, Any]):
-    """Possibly warn against misconfigured TPU chip configuration."""
-    num_tpus = resources.get(ray_constants.TPU, 0)
-    if num_tpus not in ray_constants.TPU_VALID_CHIP_OPTIONS:
-        logging.warning(
-            f"The number of requested 'TPU' was set to {num_tpus} which "
-            "is not a supported chip configuration. Supported configs: "
-            f"{ray_constants.TPU_VALID_CHIP_OPTIONS}"
-        )
-
-
-def _validate_accelerators(options: Dict[str, Any]):
```

Comment:
Should we keep this logic? Seems like it is removed

Reply:
The old logic was to disallow specifying multiple accelerator resources for a single task. But given they use the different resource names, there is no reason why Ray cannot support it.